### DISCUSSION TITLE
Adaptive Speed should be incompatible with Autoplay

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#19174] [Q&A] [answered]
If you're watching a replay with Autoplay, the game will hit every note perfectly on time, which means that even with Adaptive Speed enabled, the speed of the map will never change. Because of this, I feel like it would make much more sense if both of these mods were just incompatible with each other to begin with. If you want to adjust the initial rate of a song while using Autoplay, you can always use HT/DT or Wind Up/Down to do that instead of using Adaptive Speed.

**Comment by NotGumballer91** [comment, marked as answer]
#19195
